### PR TITLE
xdman: Change description

### PR DESCRIPTION
**Pull request by ppvnf**
Change the description according to its Github page (https://github.com/subhra74/xdm)

Relates to ScoopInstaller/Versions#735

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/xdman.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "version": "7.2.11",
-    "description": "A powerful FOSS downloader supports protocols as HTTP,HTTPS,FTP,MPEG-DASH,HLS,HDS and features as seamlessly integration with browsers, monitoring and capture, broken/dead tasks resumption.",
+    "description": "Xtreme Download Manager (XDM) is a powerful tool to increase download speeds up to 500%, save videos from popular video streaming websites, resume broken/dead downloads, schedule and convert downloads.",
```
Comment (niheaven) [inline bucket/xdman.json:3]:
```suggestion
    "description": "A powerful tool to increase download speeds up to 500%, save videos from popular video streaming websites, resume broken/dead downloads, schedule and convert downloads.",
```

Comment (ppvnf) [reply bucket/xdman.json:3]: I thought having "Xtreme Download Manager" in the description would make the meaning of XDM clearer

Comment (niheaven) [reply bucket/xdman.json:3]: Okay, keep 'Xtreme Download Manager' but remove '(XMD)'

Comment (ppvnf) [reply bucket/xdman.json]: Done

**Review by niheaven — APPROVED** [review]

**Pull request merged**